packages/mongodb-chatbot-ui/src/App.tsx

```diff
@@ -45,6 +45,7 @@ function App() {
           onClose={() => {
             console.log("Docs Chatbot closed");
           }}
+          maxInputCharacters={3000}
```

Comment:
Fine to set here, but let's also update `DEFAULT_MAX_INPUT_CHARACTERS` in `Chatbot.tsx`